examples/configs-demo/Cargo.lock

```diff
@@ -155,7 +155,7 @@ checksum = "49874b5167b65d7193b8aba1567f5c7d93d001cafc34600cee003eda787e483f"
 [[package]]
 name = "wit-bindgen-gen-core"
 version = "0.2.0"
-source = "git+https://github.com/bytecodealliance/wit-bindgen?tag=v0.2.0#cb871cfa1ee460b51eb1d144b175b9aab9c50aba"
+source = "git+https://github.com/fermyon/wit-bindgen-backport#b97517c7bfcc82f3cd6e730270d40cd4781aa789"
```

Comment:
Doesn't wit-bindgen tag v0.2.0 work anymore for slight applications?

Reply:
Isn 't this from that change you made at #344 ?

Reply:
Yes my bad. I wonder why this PR changes the Cargo.lock. Was it that I forgot to run `cargo update`?